test/CodeGeneration/CUDACtorDtor.C

```diff
@@ -36,7 +36,7 @@ for(auto &I : *M1){
 
 auto M2 = gCling->getLatestTransaction()->getModule();
 
-// The last module should be another, because of the for loop.
+// The two modules should have different names, because of the for loop.
```

Comment:
quick question: *which* for loop? I just don't find it :)

Reply:
https://github.com/SimeonEhrig/cling/blob/52c38ec2366308eb874431d8d3e7e176dd897f36/test/CodeGeneration/CUDACtorDtor.C#L26-L35

Reply:
Yes, but the names are not different be cause you read the name in the loop for M1 but they are different because you *create* two modules - with two calls to `getModule()`. Or do I get that wrong?